.github/workflows/tests.yml

```diff
@@ -64,15 +64,19 @@ jobs:
                   key: build-output-${{ github.run_id }}
             - name: Build all
               run: npm run build && npm run build.debug
+            - name: Fetch fonts
+              run: npm run fetch-fonts
             ###
             ###  Run clean-tree following the build - no changes should be present
             ###
             - name: Clean tree (build)
               run: npm run test.clean-tree
             - name: Install Playwright Browsers
               run: npx playwright install --with-deps
-            - name: Run Playwright (integration) tests
-              run: npm run test.int
+            - name: Run Playwright tests (none-screenshots)
+              run: npm run test.int -- --grep-invert @screenshots
+            - name: Run Playwright tests (screenshots only)
+              run: npm run test.int -- --grep @screenshots
```

Comment:
Run them separately to make failures easier to trace. We should potentially run these in parallel to speed up CI, but we can follow up with that